backend/src/sql_migration.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import (
+    Table,
+    Column,
+    Integer,
+    String,
+    Float,
+    Date,
+    ForeignKey,
+    create_engine,
+)
+from sqlalchemy.orm import sessionmaker
+from src.sql_service import Base, User, Account, Transaction
+
+## Testing only: drop all tables and recreate them based on sql_service schemas ##
+## Not for migration yet ##
+
+
+def main():
+    config = {
+        "DATABASE_CONNECTION_STRING": "sqlite:///../../databases/finance_tracker.db"
+    }
+    engine = create_engine(config["DATABASE_CONNECTION_STRING"])
+    Session = sessionmaker(bind=engine)
```

Comment:
Is this session creation necessary for this script to work?

Reply:
yeah I think so

Reply:
wait nm